docker/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian:8.2
```

Comment:
May I suggest either `debian:8` or `debian:jessie` instead of locking it down to a specific point-release. That would ensure security updates, and I believe binary compatibility would be maintained. But I'm not that familiar with deploying Haskell programs.